jvm/src/it/scala/com/cibo/evilplot/WriteOutDemoPlots.scala

```diff
@@ -6,35 +6,67 @@ import java.nio.file.{Files, Paths}
 import com.cibo.evilplot.demo.DemoPlots
 import javax.imageio.ImageIO
 import org.scalatest.{FunSpec, Matchers}
+import com.cibo.evilplot.geometry.Drawable
+import scala.util.Try
 
 class WriteOutDemoPlots extends FunSpec with Matchers {
 
   val plots = Seq(
```

Comment:
```suggestion
// Demo Plot Name -> Successful Plot sh1 hash
  val plots = Seq(
```

Reply:
Starting the file this way is somewhat jarring because I haven't yet encountered the details that explain what this data-structure is.

Conceptually I believe this is really Map[ Plot, Hash], but you could just as easily clarify things with a brief comment.